### PR TITLE
fixes #27103 - bump npm diff package version

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "classnames": "^2.2.5",
     "datatables.net": "~1.10.12",
     "datatables.net-bs": "~1.10.12",
-    "diff": "~3.0.0",
+    "diff": ">=4.0.1",
```
Comment (ohadlevy) [inline package.json:79]: https://github.com/theforeman/foreman/pull/6853

**Comment by ohadlevy** [comment]
bump, this is another CVE:

WS-2018-0590 
high severity
Vulnerable versions: < 3.5.0
Patched version: 3.5.0
A vulnerability was found in diff before v3.5.0, the affected versions of this package are vulnerable to Regular Expression Denial of Service (ReDoS) attacks.

File: package.json (hunk 1 above)
Comment (ekohl) [reply package.json:79]: https://github.com/theforeman/foreman-packaging/pull/3857 was intended here

**Comment by ekohl** [comment]
[test foreman]

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
After merging this I realized this breaks because we have unidiff which depends on `< 4`. https://github.com/theforeman/foreman-packaging/pull/3902 works around this because https://github.com/mvoss9000/unidiff/pull/4 hasn't seen any updates.

Note that with the new vendor JS bundle we will no longer be able to work around this and remain vulnerable.

**Pull request merged**